docs/running-rules.md

```diff
@@ -9,44 +9,49 @@ import MoreHelp from "/src/components/MoreHelp"
 
 ## Introduction
 
-Rules are specific patterns based on which Semgrep reports findings in code. These findings may help you to catch issues of security, performance, correctness, and other bugs. Rules are stored in open-source Semgrep Registry that enables you to scan code without need to write anything custom.
+Semgrep reports findings in code based on rules, which use specific patterns to match code. When Semgrep matches a portion of code, it is reported as a finding. Findings may help you to catch issues of security, performance, correctness, and best practices. Rules are stored in an open-source [Semgrep Registry](https://github.com/returntocorp/semgrep-rules) that enables you to scan code without the need to write anything custom. The list below covers different kinds of Semgrep rules:
 
-Contribute to the registry by writing your own rules and adding them to the <a href="https://github.com/returntocorp/semgrep-rules" target="_blank">Semgrep rules repository</a>.
+- Existing [Semgrep Registry rules](#running-semgrep-registry-rules) or [rules contributed to the Semgrep Registry by you](https://semgrep.dev/docs/contributing/contributing-to-semgrep-rules-repository/) to find out how to add new rules.
+- [Local rules](#creating-and-using-local-rules):
+  - One-off [ephemeral rules](#ephemeral-rules) passed into the command line.
+  - [YAML-defined rules](#yaml-defined-rules).
+- A combination of [local rules and Semgrep Registry rules](#running-multiple-rules-simultaneously). 
 
-<div className="lang-container" style={{marginBottom: '20px'}}>
-  <iframe width="900" height="400" frameBorder="0" src="https://dashboard.semgrep.dev/metric/semgrep-rules.num/graph"></iframe>
-</div>
+You can run all rules on your code completely locally or continuously in your Source Code Management (SCM) service (such as GitHub or GitLab) with Semgrep in CI. For more information, see [Semgrep CI overview](semgrep-ci/overview.md).
 
-You can run existing and custom Semgrep rules locally with the Semrgep command line tool or continuously with Semgrep CI. See [Getting started](../getting-started/) for their respective installation and setup.
+## Running Semgrep Registry rules locally
 
-## Run Registry rules
+You can test how Semgrep automatically surveys languages and frameworks and run recommended rules for your source code with the following command (substitute the `PATH/TO/SRC` with path to your source code):
```

Comment:
This is not accurate. As of now we do not run surveys to recommend rules.

Reply:
@brendongo what is the logic to determine the language or framework now? @brendongo What do we do instead of surveys?

Reply:
We send the same rules no matter what